controller/src/main/java/io/pravega/controller/store/stream/AbstractStreamMetadataStore.java

```diff
@@ -564,12 +564,12 @@ public OperationContext createContext(String scope, String name) {
         return withCompletion(getStream(scopeName, streamName, context).checkTransactionStatus(txId), executor);
     }
 
-    @Override
+    @VisibleForTesting
```

Comment:
Does this no longer override anything?

Reply:
no.. the transactions are comitted in a batch.. so the commit transaction api on the store interface was only used in tests.. so i removed it from the interface and marked this method visible for testing